client/lcd/certificates.go

```diff
@@ -145,7 +145,9 @@ func fingerprintForCertificate(certBytes []byte) (string, error) {
 		return "", err
 	}
 	h := sha256.New()
-	h.Write(cert.Raw)
+	if _, err := h.Write(cert.Raw); err != nil {
```

Comment:
Nit: sha256's write actually doesn't err. This is perfect for a `nolint` with a comment.

See: https://golang.org/src/crypto/sha256/sha256.go?s=4118:4138#L203

Reply:
Ah! Nice catch, didn't know about that, thanks! Nonetheless, certificates.go will be gone shortly - we intend to remove the `--secure` option from the REST server altogether.